configuration/pipeline_config.json

```
@@ -122,6 +122,11 @@
   "ProjectEndDate": "2100-01-01T00:00:00+03:00",
   "FilterTestMessages": true,
   "MoveWSMessages": true,
+  "AutomatedAnalysisConfiguration": {
```

```suggestion
  "AutomatedAnalysis": {
```
(It's obvious that this is configuration)

---

updated thanks